ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextAnchorViewManager.java

```diff
@@ -42,16 +42,19 @@
     Spacing.ALL, Spacing.LEFT, Spacing.RIGHT, Spacing.TOP, Spacing.BOTTOM,
   };
   private static final String TAG = "ReactTextAnchorViewManager";
+  private Integer mNumberOfLines = ViewDefaults.NUMBER_OF_LINES;
```

Comment:
NIT: can you use int instead of Integer here?

Reply:
Thanks a lot @mdvacca I added the change with commit https://github.com/facebook/react-native/pull/31064/commits/f602cd9e90cc45b52fa64a43013c4a5c44370f65 I wish you a good day